app/views/welcome/quotes.html.erb

```diff
@@ -0,0 +1,8 @@
+<h3><%= @quotes[:total] %> total results found for "<%= @search %>"</h3>
```

Comment:
We never want to call hashes in our view. That is why we create POROs so that we can have objects to use here. I would spend some time over the weekend to see how you can get objects to replace these hashes.